spec/spec_helper.rb

```diff
@@ -1,8 +1,31 @@
-require 'rspec-puppet'
+require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
+include RspecPuppetFacts
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
Interesting, that file exists in moduleroot of our configs, I assumed that meant it pushed every time. Learn something new every day.